addons/io_hubs_addon/debugger.py

```diff
@@ -77,8 +77,10 @@ def export_scene(context):
         'use_active_collection': export_prefs.use_active_collection,
         'export_apply': export_prefs.export_apply,
         'export_force_sampling': False,
-        'use_active_scene': True
     }
+    if bpy.app.version >= (3, 2, 0):
```

Comment:
Just curious does this do any harm if < 3.2.0? Does it crash?

Reply:
The `use_active_scene` export option isn't present in the glTF add-on in Blender versions under 3.2, so trying to include it causes the export to fail in those versions.